src/components/attendee/Card/Card.module.scss

```diff
@@ -19,7 +21,11 @@
 @each $class, $size in layout.$card-sizes {
```

Comment:
[NOTE][OOS]
Just like above, we can probably uniform these sizes for all generic components (Button, Input, Toggle etc.) and then just provide the meaningful styles in the SCSS file.